src/NATS.Client.Core/Internal/SslClientAuthenticationOptionsExtensions.cs

```diff
@@ -30,9 +30,9 @@ public static SslClientAuthenticationOptions LoadClientCertFromPem(this SslClien
         // https://github.com/dotnet/runtime/blob/380a4723ea98067c28d54f30e1a652483a6a257a/src/libraries/System.Net.Security/tests/FunctionalTests/TestHelper.cs#L192-L197
         if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
         {
-            var ephemeral = leafCert;
-            leafCert = new X509Certificate2(leafCert.Export(X509ContentType.Pfx));
-            ephemeral.Dispose();
+            var ephemeral = new X509Certificate2(leafCert.Export(X509ContentType.Pfx));
+            leafCert.Dispose();
+            leafCert = ephemeral;
```

Comment:
isn't this essentially the same thing? unless i am missing something.

Reply:
You're right, I'll rollback this change. Thanks!